go.mod

```diff
@@ -1 +1,3 @@
 module github.com/alexandrevicenzi/go-sse
+
+go 1.12
```

Comment:
Could it be 1.11 here?

Reply:
It was autogenerated by the go tool.

Reply:
Because you are using 1.12, if you use the docker image inside it's 1.11 and would not change anything.

Reply:
Oh sure,I had forgot bout the docker.Let me change it.

Reply:
On hindsight I could just update the docker-compose script to pull a go1.12 image.